scenarios/action_recognition/02_training_hmdb.ipynb

```diff
@@ -0,0 +1,965 @@
+{
```

Comment:
<p>Really like how this and the VideoDataset follows the detection naming and syntax.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microsoft/computervision-recipes/pull/477/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>